### DISCUSSION TITLE
Read shared config from package.json

### DISCUSSION THREAD

**Discussion by danielbayley** [biomejs/biome#4532] [Ideas]
Seems silly to have yet another root `biome.json` file, just to use `extends`:
~~~ jsonc
// biome.json
{
  "extends": ["@org/shared-configs/biome"]
}
~~~

Surely it would be better, in this case, to instead read from a `biome` key from `package.json`:
~~~ jsonc
// package.json
"biome": {
  "extends": ["@org/shared-configs/biome"]
},
~~~

> [!NOTE]
> To clarify, I am only suggesting to allow `extends` for this purpose, not the entire config.

**Comment by ematipico** [comment]
We decided we don't want to pollute the package.json with our configuration.

Considering the following:
- Other tools do that too
- Biome configuration is big and it grows over time 

package.json could risk to become a massive file with too much information.

Plus, you would loose the completion given by the schema

**Reply by danielbayley** [reply]
> We decided we don't want to pollute the package.json with our configuration.

@ematipico To clarify, I am only suggesting to allow `extends`, not the entire config!
